### PR TITLE
chore: Add tests for Vulnerability analysis API

### PR DESCRIPTION
**Pull request by sawood14012**
# Description

Added Tests for the new API which will be used by CLAIR / QUAY 

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by jparsai — APPROVED** [review]

File: integration-tests/features/steps/vulnerablity-analysis.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+
+def get_token_call_url(context):
+    """Construct url for get-token call."""
+    return urljoin(context.core_v2_api_url,
+                   '/api/v2/get-token')
+
+
+def perform_Vuln_batch_call(context, use_user_key, packages_file):
+    """Perform Batch Call for vulnerability-analysis."""
```
Comment (yzainee) [inline integration-tests/features/steps/vulnerablity-analysis.py:25]: how many reqs are you going to send in total for this?

Comment (sawood14012) [reply integration-tests/features/steps/vulnerablity-analysis.py:25]: Somewhere around 10 - 11 including all the cases 
Only positive cases 5

**Pull request merged**